helical/models/uce/model.py

```diff
@@ -116,12 +116,8 @@ def process_data(self,
                                                                         species=[species],
                                                                         embedding_model=embedding_model,
                                                                         embeddings_path=Path(files_config["protein_embeddings_dir"]))
-        
         # TODO: What about hv_genes? See orig.
-        if scipy.sparse.issparse(filtered_adata.X):
-            gene_expression = np.asarray(filtered_adata.X.todense())
-        else:
-            gene_expression = np.asarray(filtered_adata.X)
+        gene_expression = adata.X.toarray()
```

Comment:
The original had this [here](https://github.com/snap-stanford/UCE/blob/main/data_proc/data_utils.py#L224)
![image](https://github.com/user-attachments/assets/6975d648-859f-403d-9946-da105055a00b)
and
![image](https://github.com/user-attachments/assets/46366ea3-c6ea-44ea-9afb-d25bfb5d07d4)

Reply:
As I read this, they convert adata.X to numpy (in the function), then to torch (in the return line) and then back to numpy outside the function. 
So I would do it directly to numpy. Looking at the `.toarray()` function from `anndata.X`, it returns a numpy array.
![image](https://github.com/user-attachments/assets/000ce7e7-d3bb-4117-9cf8-93ca048a1319)
So I propose to use this directly.